### DISCUSSION TITLE
Token Classification / Named Entity Recognition

### DISCUSSION THREAD

**Discussion by davanstrien** [AI4LAM/TeachingAndLearning#7] [GLAM AI Tasks]
This is a discussion for the Token Classification task:

> Token classification is a natural language understanding task in which a label is assigned to some tokens in a text. Some popular token classification subtasks are Named Entity Recognition (NER) and Part-of-Speech (PoS) tagging. NER models could be trained to identify specific entities in a text, such as dates, individuals and places; and PoS tagging would identify, for example, which words in a text are verbs, nouns, and punctuation marks. [source](https://huggingface.co/tasks/token-classification)

**Comment by davanstrien** [comment]
AI4LAM community calls on this topic: 

## Introduction to NER 
*Javier de la Rosa, National Library of Norway*
[![](https://img.youtube.com/vi/XkwoMVOs6Sw/0.jpg)](https://www.youtube.com/watch?v=XkwoMVOs6Sw)


## NER and Knowledge Graphs in the Heritage Connector Project. 
*Kalyan Dutia, Science Museum Group, UK* 
[![](https://img.youtube.com/vi/OrvQkghi8mU/0.jpg)](https://www.youtube.com/watch?v=OrvQkghi8mU?t=79)

**Comment by saracarl** [comment]
Here's a [write-up on experiments in identifying entities in Ancient Greek and Classical Arabic](https://content.fromthepage.com/machine-learning-to-extract-entities-from-ancient-greek-and-other-languages/) using Spacy from a prosopography hackathon at the University of Vienna in 2019.
